### ISSUE TITLE
Are you plan to implement NodeJS lib?

### ISSUE THREAD

**Issue by vdanyliv** [amzn/selling-partner-api-docs#32]
Hello, thanks for the incredible updates.
I plan to use a new API for my project, and I wonder if you plan to implement lib for nodejs? (I asked you because I plan to start working on that)

**Comment by charliecode** [comment]
@vdanyliv I was wondering the same thing. I can't speak for Amazon but it appears so far they've provided libs in the same languages as they did for MWS, which does not include Node. However, they have provided templates which can be used with swagger-codegen to generate client libraries as is stated on the main page for this repo where it says -

"The selling-partner-api-models/clients directory contains a Java library and a C# library with mustache templates for use with swagger-codegen to generate client libraries with authentication and authorization functionality included. The templates are located in resources/swagger-codegen."

**Comment by charliecode** [comment]
@dominickmalzone Thank you. Best way to contact you?

**Comment by dominickmalzone** [comment]
You can email me @chapmanjw - its my github username @ gmail

**Comment by charliecode** [comment]
@dominickmalzone Just sent you an email. In the event I typo'd the email address and it doesn't reach you, please let me know.

**Comment by beastmike1** [comment]
Hi @seanevan there is any roadmap or hint date about when it will be available it would be highly appreciated so we could plan the development of our software. Thanks.

**Comment by seanevan** [comment]
Hi @beastmike1, unfortunately we don't have a timeline that I can share as of yet. Would I be correct in understanding that the lack of a NodeJS client library is blocking your adoption of SP-API? I ask because that type of feedback helps our product team with prioritization decisions.

Thanks,
Sean Evans
Manager, Support Engineering

**Comment by beastmike1** [comment]
Yes, we use NodeJS in our products and not having the client is blocking the adaptation because for us it would be a lot of extra work to change to another language. It would be really good to at least have the templates for nodeJS.
